json/actions.json

```diff
@@ -527,5 +527,319 @@
       "professors": ["maurizio.gabbrielli"],
       "telegram": "ai_unibo"
     }
+  },
+  "decision_making": {
+    "type": "course",
+    "data": {
+      "description": "Tutto su decision making",
```

Comment:
I titoli degli insegnamenti in Italiano hanno la lettera maiuscola (solo sulla prima parola), quindi aggiornerei i valori dei campi `description` che hai aggiunto.

In inglese invece, i titoli (inclusi quelli degli insegnamenti) hanno le maiuscole su tutte le parole che non siano né all'inizio della frase né usate come articoli/preposizioni/congiunzioni di poche lettere. Scegli tu se vuoi adottare le convenzioni inglesi per gli insegnamenti in inglese o fingere che sia comunque italiano.